src/D2L.SQL.Validation/Language/SqlGrammar.cs

```diff
@@ -20,9 +20,6 @@ public SqlGrammar()
 			number.DefaultIntTypes = new TypeCode[] { TypeCode.Int64 };
 			var string_literal = new StringLiteral( "string", "'", StringOptions.AllowsDoubledQuote );
 			var Id_simple = TerminalFactory.CreateSqlExtIdentifier( this, "id_simple" );
-			// column names, unlike other ids, may start with a numeral
-			var columnId = TerminalFactory.CreateSqlExtIdentifier( this, "columnId" );
-			columnId.AllFirstChars += "0123456789";

```

Comment:
Also Phoenix grammar doesn't allow digits at the start of a name, so we're not breaking anything that used to work.